src/main/java/io/github/externschool/planner/entity/Authority.java

```diff
@@ -1,7 +1,7 @@
 package io.github.externschool.planner.entity;
 
 import javax.persistence.*;
-import java.util.Objects;
+import java.util.*;
```

Comment:
please do not use wildcard imports (use your IntelliJ IDEA to set: Settings → Editor → Code Style → Java → Imports → Class count to use... → any big number like 99)